base/float.jl

```diff
@@ -596,7 +596,7 @@ uabs(x::BitSigned) = unsigned(abs(x))
 The result of `n` iterative applications of `nextfloat` to `x` if `n >= 0`, or `-n`
 applications of `prevfloat` if `n < 0`.
 """
-function nextfloat(f::IEEEFloat, d::Integer)
+function nextfloat(f::AbstractFloat, d::Integer)
```

Comment:
So this one needs to stay as it was since the code requires IEEE layout. The other one just calls `nextfloat` so it can be more generic and will fail if there is not `nextfloat` method.